taxonomy-editor-frontend/src/constants.js

```diff
@@ -17,3 +17,33 @@ function taxonomyApiUrlFromUi(location) {
 }
 export const API_URL = taxonomyApiUrlFromUi(window.location);
 export const ENTER_KEYCODE = 13;
+export const TAXONOMY_NAMES = [
```

Comment:
please add a comment of where this taxonomies come from, to have an idea of when they need to be updated or where to check for that

Reply:
I added a wiki doc for a list of taxonomies currently in Open Food Facts